### DISCUSSION TITLE
Welcome to 2anki.net Discussions!

### DISCUSSION THREAD

**Discussion by aalemayhu** [2anki/server#248] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ricardovictordev** [comment]
I have some note types that have more fields than basic and cloze. How could I use this tool?

**Reply by aalemayhu** [reply]
You can build custom note types with as many fields as you need at https://2anki.net/templates — add/rename fields, edit the front, back, and CSS, preview, and export
   to .apkg. Saved templates can also drive the Notion conversion itself.
